extras/github/docker.py

```diff
@@ -160,7 +160,7 @@ def overwrite_version(base_version: str):
     with open('BUILD_VERSION', 'w') as file:
         if base_version.startswith('v'):
             base_version = base_version[1:]
-        file.write(base_version + '\n')
+        file.write(base_version)
```

Comment:
Should we also add a `.strip()` in the `hathor.version._get_build_version()`?

Reply:
Yeah, I think so

Reply:
Great! Thanks!